code/HabitTracker/app/src/main/java/com/example/habittracker/HabitEvent.java

```diff
@@ -44,7 +44,7 @@ public HabitEvent () {
      */
     public HabitEvent (String habit, String eventId ,String comment, ArrayList<Integer> startDate, String location, String image) {
 
-        this.userId = "user1";
+        this.userId = "John_test_user";
```

Comment:
It was my understanding that the login thing is up and running rn?
You should be able to change this to a getCurrentUser call (I recall the function was called something along those lines?)

Reply:
Fixed

Reply:
fixed